tagging/taggers.py

```diff
@@ -14,30 +14,27 @@ def _get_program_version(container, program: str) -> str:
 
 def _get_env_variable(container, variable: str) -> str:
     env = DockerRunner.run_simple_command(
-        container,
-        cmd="env",
-        print_result=False
+        container, cmd="env", print_result=False
     ).split()
     for env_entry in env:
         if env_entry.startswith(variable):
-            return env_entry[len(variable) + 1:]
+            return env_entry[len(variable) + 1 :]
     raise KeyError(variable)
 
 
 def _get_pip_package_version(container, package: str) -> str:
     VERSION_PREFIX = "Version: "
     package_info = DockerRunner.run_simple_command(
-        container,
-        cmd=f"pip show {package}",
-        print_result=False
+        container, cmd=f"pip show {package}", print_result=False
```

Comment:
Was better

Reply:
The new or old was better? I won't add a trailing comma to retain previous behavior as I guess you liked the new way and I'm not especially opinionated about these style decisions.

Reply:
I prefer one of two ways:

```
func(a, b, c, d)
```

or
```
func(
    a,
    b,
    c,
    d
)
```

The second option is also ok for me, when there is a comma.
I don't really like the mix of two of these.

Reply:
Reading another comment I now interpret your comment as it was better before and added a trailing comma.

Reply:
Yes, when I say "was better", I mean it was better before the changes by black.
I need to practice English from time to time :)